pulser-core/pulser/backend/qpu.py

```diff
@@ -34,6 +36,7 @@ def run(self, job_params: list[JobParams] | None = None) -> RemoteResults:
                 is parametrized, the values for all the variables necessary
                 to build the sequence must be given in it's own mapping, for
                 each job, under the 'variables' field.
+            wait: Whether to wait until the results become available.
```

Comment:
Maybe add a bit more information: "Whether wait until the results of the jobs become available. If set to False, the call is non-blocking and the results must be checked using the `results()` property."

Reply:
Discussing with @LucasGitQ, we propose "Whether wait until the results of the jobs become available. If set to False, the call is non-blocking and the obtained results must be checked using their `status` property."

Reply:
Thanks for the suggestions!